dnsaas/api/v2/serializers.py

```diff
@@ -155,6 +171,18 @@ def _ensure_owner_is_set(self):
                 ]
             })
 
+    def _validate_service(self, attrs):
+        if not settings.REQUIRED_SERVICE_FIELD or self.instance:
```

Comment:
- REQUIRED_SERVICE_FIELD == False: service is not required
- self.instance is not None: it's record update so we don't need to check service